resource-sharing-partners/src/main/java/no/sikt/basebibliotek/BasebibliotekConverter.java

```diff
@@ -0,0 +1,84 @@
+package no.sikt.basebibliotek;
+
+import static nva.commons.core.StringUtils.isEmpty;
+import jakarta.xml.bind.JAXBElement;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.Locale;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import javax.xml.datatype.XMLGregorianCalendar;
+import no.nb.basebibliotek.generated.BaseBibliotek;
+import no.nb.basebibliotek.generated.Record;
+import nva.commons.core.JacocoGenerated;
+
+public class BasebibliotekConverter {
+    
+    private static final DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd", Locale.US);
+    private static final String NNCIP_URI_FIELD_NAME = "nncip_uri";
+
+    @JacocoGenerated
+    public BasebibliotekConverter() {
+    }
+
+    public static List<BaseBibliotekBean> convertBasebibliotekToBaseBibliotekBean(BaseBibliotek baseBibliotek) {
+        return baseBibliotek
+                   .getRecord()
+                   .stream()
+                   .map(BasebibliotekConverter::convertRecordToBasebibliotekBeanIfBibNrAndLandkodeIsSet)
+                   .flatMap(Optional::stream)
+                   .collect(Collectors.toList());
+    }
```

Comment:
Er det det hva vi gjør her? Converter BaeBibliotek i BaseBibliotekBean? Hvorfor gjør vi det egentlig? 
BaseBibliotekBean er en enklere versions av record-objekt BaseBibliotek, ikke sant. Kunne vi ikke bare bruke BaseBibliotek direkte og har noen hjelpe methoder i en Utils-klasse istedenfor å sette opp en eget bean?

Reply:
Tja, si det. 
Ser at basebibliotek bean inneholder noe funksjonalitet for å kalkulere om et bibliotek er åpent der og da.

Reply:
Dkal undersøke hva basebibliotekBean blir brukt til senere i løypa i gammel RSP, siden gammel RSP både lager basebibliotekBean og benytter den senere.